Clicker/Views/MultipleChoice/AddMoreOptionCell.swift

```diff
@@ -27,8 +33,17 @@ class AddMoreOptionCell: UICollectionViewCell {
         setupViews()
     }
     
+    func configure(with: PollBuilderMCOptionSectionControllerDelegate) {
```

Comment:
rename this to `func configure(with delegate: PollBuilderMCOptionSectionControllerDelegate)`